Gulpfile.js

```diff
@@ -107,6 +112,7 @@ const projects = [
         cwd: "packages/table/",
         dependencies: ["core"],
         copy: false,
+        isotest: true,
```

Comment:
The mapping is indeed 1:1. If it looks like it will stay that way, we can merge them. `isotest` has a compile/build dependency which `karma` does not. If we combine the tasks, the combined task will also have this dependency. If you can live with this, I think the merged task will be an improvement. (Def think `karma` and `isotest` are not the best names)